flask_security/datastore.py

```diff
@@ -165,23 +169,17 @@ def __init__(self, db, user_model, role_model):
         SQLAlchemyDatastore.__init__(self, db)
         UserDatastore.__init__(self, user_model, role_model)
 
-    def get_user(self, identifier):
```

Comment:
disclaimer: I don't have the full context, but personally I would keep the old method for backwards compatibility. To be confirmed before changing

Reply:
Makes sense, we'll probably limit the use of the original method (to use one of email/ID)